app/views/home/index.html.haml

```diff
@@ -23,214 +28,146 @@
     - if current_user
       - if current_user.admin
         = link_to manage_dashboard_index_path do
-          .button.button-outline MANAGE
+          .button MANAGE
       - else
         = link_to questionnaires_path do
-          .button.button-outline MY ACCOUNT
+          .button MY ACCOUNT
     - elsif Rails.configuration.hackathon['registration_is_open']
       = link_to new_user_registration_path do
-        .button.button-outline SIGN UP TODAY!
+        .button REGISTER NOW
     - if Rails.configuration.hackathon['event_is_over']
       = link_to "https://brickhack3.devpost.com/submissions/" do
-        .button.button-outline SUBMISSIONS
+        .button SUBMISSIONS
       = link_to URI.escape("mailto:hello@brickhack.io?subject=BrickHack Organizer Interest&body=Hey there! I'm interested in helping organize BrickHack. How can I help?") do
         .button.button-outline HELP ORGANIZE
-    - else
-      %br
-      = link_to volunteer_link do
-        .button.button-outline VOLUNTEER
-      %br
-      = link_to mentor_link do
-        .button.button-outline MENTOR
-
-.white-section.diagonal#about
-  .home-content.about-gallery
-    .about
-      %h1 RIT's Premiere Hackathon
+  .split-right#home-right
+    .half-img#home-img
+      = image_tag 'hackathon.jpg'
+.page.split.about-gallery
+  .split-left.gallery-left
+    = image_tag 'gallery-img.png'
+    .button-wrapper
+      = link_to 'Full Gallery', '/gallery/', class: 'button button-overlay'
+
+  .split-right.gallery-right
+    %h2
+      Brick City's 
+      .orange
+        Premiere Hackathon
+    .gallery-text#about
       %p
         BrickHack brings
-        %strong designers, developers,
-        and
-        %strong all sorts of makers
-        together for 24 hours to create the impossible!
+        designers, developers, and all sorts of makers together for 24 hours to create the impossible!
```

Comment:
nit: this language was tied to the "create the impossible" campaign last year; without the accompanying graphics from before, this might come off a bit extreme to someone with little context of what hackathons are